docs/installation.rst

```diff
@@ -84,41 +82,66 @@ There are no **mandatory** external dependencies. However, some optional feature
 .. note:: You can install these additional components at any time -- before or after installing PyMuPDF. PyMuPDF will detect their presence during import or when the respective functions are being used.
 
 
-Installation from source without using an sdist
+Build and install from local PyMuPDF checkout and optional local MuPDF checkout.
 ---------------------------------------------------------
 
-* First get a PyMuPDF source tree:
+* Install C/C++ development tools as described above.
+
+* Enter a Python venv and update pip, as described above.
+
+* Get a PyMuPDF source tree:
 
-  * Clone the git repository at https://github.com/pymupdf/PyMuPDF,
-    for example::
+  * Clone the PyMuPDF git repository:
 
-      git clone https://github.com/pymupdf/PyMuPDF.git
+        git clone https://github.com/pymupdf/PyMuPDF.git
 
   * Or download and extract a `.zip` or `.tar.gz` source release from
     https://github.com/pymupdf/PyMuPDF/releases.
 
-* Install C/C++ development tools and SWIG as described above.
 
-* Build and install PyMuPDF::
+* Build and install PyMuPDF:
```

Comment:
I think the `::` denotes that the tab indented stuff appears as <code>

Reply:
Yes, i was trying to avoid this, but it doesn't work, so have fixed to use `::`.